src/Rings/mpolyquo-localizations.jl

```diff
@@ -183,13 +187,37 @@ For ``L = (𝕜[x₁,…,xₙ]/I)[S⁻¹]`` this returns ``𝕜[x₁,…,xₙ]/I
 """
 underlying_quotient(L::MPolyQuoLocalizedRing) = L.Q
 
+## 3 more signatures for compatibility to make quotient_ring agnostic
+underlying_quotient(L::MPolyQuo) = L
+
+@attr MPolyQuo function underlying_quotient(L::MPolyRing)
```

Comment:
I don't think that this will be very helpful because it creates a new ring every time it is called. 
Maybe being too agnostic is not beneficial here?

```
julia> P,(x,y) = PolynomialRing(QQ,[:x,:y])
(Multivariate Polynomial Ring in x, y over Rational Field, fmpq_mpoly[x, y])

julia> I = ideal(P,[P(0)])
ideal(0)

julia> quo(P,I)[1] === quo(P,I)[1]
false

julia> A = quo(P,I)[1]
Quotient of Multivariate Polynomial Ring in x, y over Rational Field by ideal(0)

julia> B = quo(P,I)[1]
Quotient of Multivariate Polynomial Ring in x, y over Rational Field by ideal(0)

julia> A(1)
1

julia> A(1)*B(1)
ERROR: wrong parents
```

Reply:
Or one could cache this?

Reply:
Stupid beginner's question: isn't this already cached by the '@attr' construction?

Reply:
True, I missed that.